client/Components/DocPages/Basics/ServerSideRendering.tsx

```diff
@@ -150,7 +150,9 @@ import App from './App.tsx';
       </p>
       <CodeBlock language="tsx" showLineNumbers={true} style={dracula}>
         {`// server.tsx
-const [_, clientJS] = await Deno.bundle('./client/client.tsx');`}
+const { files, diagnostics } = await Deno.emit('./client/client.tsx', {
+  bundle: 'esm',
+});`}
```

Comment:
This looks good, but we should update the paragraph above (line 147) to reference "Deno's native emit method" rather than bundle.

Reply:
Done